content/show/linux-unplugged/_index.md

```diff
@@ -13,15 +13,30 @@ header_image = "/images/shows/linux-unplugged.png"
 support_link = "https://jupitersignal.memberful.com/checkout?plan=52946"
 
 podverse_podcast_id = "g40Um-HP1"
-+++
 
-Your virtual Linux User Group (vLUG) with no limits, and tons of opinion. An open show powered by community LINUX Unplugged takes the best attributes of open collaboration and focuses them into a weekly lifestyle show about Linux.
+[links.twitter]
+  url = "https://twitter.com/LINUXUnplugged"
+
+[links.email]
+  url = "https://linuxunplugged.com/contact"  
+
+[links.shownotes]
+  url = "https://linuxunplugged.com/"
+
+[links.reddit]
+  url="https://www.reddit.com/r/linuxunplugged/"
+
+[links.matrix]
+  url="https://matrix.to/#/+lup:jupiterbroadcasting.com"
+
+[links.mumble]
+  url="https://linuxunplugged.com/mumble"
+
+[links.blog]
+  url="https://linuxunplugged.com/articles" 
+
+[links.youtube]
+  url="https://www.youtube.com/watch?v=CgTyhkI_qkE&list=PLUW3LUwQvegxVckkyK5dyyARdfjwGY-cw"
```

Comment:
same here re playlist itself vs video+playlist

should look more like you did in LAN: https://www.youtube.com/playlist?list=xxxxxxxx